eox_nelp/pearson_vue/pipeline.py

```diff
@@ -496,3 +498,104 @@ def raise_audit_pearson_exception(exception_dict, failed_step_pipeline):
         raise_audit_pearson_exception(failed_step_pipeline=failed_step_pipeline, exception_dict=exception_dict)
     except PearsonBaseError as exc:
         logger.error(exc)
+
+
+def extract_result_notification_data(request_data, **kwargs):  # pylint: disable=unused-argument
+    """
+    Extracts result notification data from the request.
+
+    Args:
+        request_data (dict): The dictionary containing the Result Notification request data.
+        **kwargs: Additional keyword arguments.
+
+    Returns:
+        dict: A dictionary containing extracted data including exam result, client authorization ID,
+              enrollment ID, anonymous user model ID, and anonymous user ID.
+    """
+    client_authorization_id = request_data["authorization"]["clientAuthorizationID"]
```

Comment:
Are you going to handle key error with pearsonvue exceptions?? or could be other PR

Reply:
Thanks for the reminder, I will implement that in this PR